src_c/draw.c

```diff
@@ -958,13 +956,13 @@ clip_and_draw_line_width(SDL_Surface *surf, SDL_Rect *rect, Uint32 color,
 {
     int xinc = 0, yinc = 0;
     int bounding_rect[4];
+    int original_values[4];
+    int anydrawn = 0;
+    int loop;
     bounding_rect[0] = INT_MAX;
     bounding_rect[1] = INT_MAX;
     bounding_rect[2] = 0;
     bounding_rect[3] = 0;
-    int original_values[4];
-    memcpy(original_values, pts, sizeof(int) * 4);
```

Comment:
Did you accidentally remove this memcpy?

Reply:
I just moved it a bit down just before it is used first time, but I will try moving it up now

Reply:
Also, the memcpy src and dst should be swapped?

```
     memcpy(void *restrict dst, const void *restrict src, size_t n);
```


To this:
```
memcpy(pts, original_values, sizeof(int) * 4);
```

Reply:
(ignore my last comment about swapping it)

Reply:
Actually it was probably this line, I moved it too low (after I change pts) and that could maybe cause bug.